extension/src/popup/components/manageAssets/AddAsset/index.tsx

```diff
@@ -151,8 +151,8 @@ export const AddAsset = () => {
           setVerifiedLists(verifiedTokens[0].verifiedLists);
           setAssetRows(
             verifiedTokens.map((record: VerifiedTokenRecord) => ({
-              code: record.code,
-              issuer: record.issuer,
+              code: record.code || record.contract,
+              issuer: record.issuer || record.contract,
```

Comment:
there is an edge case where a user's asset list can be missing fields but still pass the schema validation Orbit Lens created. In that case, we're adding some fallback values here

Reply:
I'll take to OL about tightening the schema rules